components/Dialogs/src/MDCAlertController.m

```diff
@@ -212,6 +212,19 @@ - (void)setButtonTitleColor:(UIColor *)buttonColor {
   }
 }
 
+- (void)setCornerRadius:(CGFloat)cornerRadius {
+  _cornerRadius = cornerRadius;
+  if (self.alertView) {
+    self.alertView.cornerRadius = cornerRadius;
+  }
+  // make sure to pass through the new shape to the dialog's tracking & shadow layers
+  MDCDialogPresentationController *dialogPresentationController =
+      self.mdc_dialogPresentationController;
+  if (dialogPresentationController) {
```

Comment:
Please remove the `nil` check since `nil` safely receives all messages.

http://google.github.io/styleguide/objcguide.html#nil-checks